packages/api-types/src/modules/dispositif.ts

```diff
@@ -275,6 +272,24 @@ export type GetDispositifResponse = {
   origin: DispositifOrigin;
 };
 
+export type GetDispositifResponse = BaseGetDispositifResponse &
+  (
+    | {
+        markdown: string;
+        what?: string;
```

Comment:
If the markdown property is present, the what  and how properties should be absent.